Chapter3_exercise#2/Chapter3_exercise#2/ViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ViewController.swift
+//  Chapter3_exercise#2
+//
+//  Created by Andreea Sburlea on 4/7/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+    @IBAction func showMessage2(sender: UIButton){
+        let emojiDict = ["👾" : "Alien", "👻" : "Ghost", "🤓" : "Nerd", "🤖" : "Robot"]
+        let wordToLookup = "👾"
+        let meaning = emojiDict[wordToLookup]
+     
+        
+        let selectedButton = sender
+        
+        if let wordToLookup = selectedButton.titleLabel?.text{
+           let meaning = emojiDict[wordToLookup]
+            
+        let alertController = UIAlertController(title: "Meaning", message: meaning, preferredStyle: UIAlertController.Style.alert)
+        
+        alertController.addAction(UIAlertAction(title: "OK", style: UIAlertAction.Style.default, handler: nil))
+            present(alertController, animated: true, completion: nil)
+        }
```

Comment:
Every line inside the if statement should be aligned and with +1 tab indentation